README.md

```diff
@@ -83,6 +83,20 @@ if variant.enabled {
 }
 ```
 
+### Available options
+
+The Unleash SDK takes the following options:
+
+| option            | required | default                   | description                                                                                                                                      |
+|-------------------|----------|---------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| unleashUrl        | yes | n/a                       | The Unleash Proxy URL to connect to. E.g.: `https://examples.com/proxy`                                                                         |
+| clientKey         | yes | n/a                       | The Unleash Proxy Secret to be used                                                                                                             | 
```

Comment:
Is this an API key from https://docs.getunleash.io/reference/api-tokens-and-client-keys#front-end-tokens or something else? Maybe add the link

Reply:
It's consistent with other SDKs. We can upgrade all of the in one go. I'll suggest this effort.